packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -369,6 +369,9 @@ export class TdsDropdown {
 
   private getValue = () => {
     const labels = this.getSelectedChildrenLabels();
+    if (!labels) {
+      return '';
```

Comment:
Any reason for returning a blank string over `null`?

Reply:
Is it not more appropiate to return empty string here, as the function is designed to return a string representation of labels?

Reply:
A text inputs value is always a string, so i think returning an empty string is ok. Null might work as well, but it converted undefined to "undefined" so 🤷‍♂️